cost/turbonomics/rightsize_databases_recommendations/aws/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## v0.3
+
+- Added `VM Age` incident field.
+- Added `Pricing Model` incident field.
+- Added `URL` incident field.
+- Added `Actual Storage Amount` incident field.
+- Added `Actual Storage Amount % Utilization` incident field.
+- Added `New Storage Amount` incident field.
```

Comment:
Maybe we could use "After Actions" instead of "New". Example: "New Size" (Remember that "Storage Amount" should be renamed to "Size"). We should check with Tatiana what should we use for the after actions items.